src/components/Accounts/ChargeFeeFine/Charge.js

```diff
@@ -292,7 +292,7 @@ class Charge extends React.Component {
         <FormattedMessage id="ui-users.charge.messageOf" />
         <strong>{`${parseFloat(a.amount).toFixed(2)}`}</strong>
         <FormattedMessage id="ui-users.charge.messageSuccessfully" />
-        {a.paymentStatus.name}
+        <strong>{(a.paymentStatus.name).toLowerCase()}</strong>
```

Comment:
This whole block, lines 290-297 should be a single translation string that accepts multiple parameters. e.g. https://github.com/folio-org/ui-users/blob/master/src/components/Loans/OpenLoans/components/OpenLoansSubHeader/OpenLoansSubHeader.js#L123